util/div-creator.js

```diff
@@ -151,16 +151,47 @@ function createAttachmentDiv(a) {
 			img.addEventListener("click", handler);
 			div.appendChild(img);
 		}
+	} else if(isVideoExt(a.name)) {
+		if(a.width > 1920 || a.height > 1080) {
+			var text = document.createElement("text");
+			text.className = "embed-errtxt attachment";
+			text.innerHTML = "Refused to load video due to it being too big.";
```

Comment:
needs `<br>Click to view` and click listener to open the video in a new tab (-> popup, links are spooky)

Reply:
Ok

Reply:
Doing it later